SPECS/mariadb/generate_source_tarball.sh

```diff
@@ -1,16 +1,100 @@
 #!/bin/bash
-if [[ $# -eq 0 ]] ; then
-    echo 'Usage: generate_source_tarball.sh <MariaDB version>'
-    echo 'Example: generate_source_tarball.sh 10.3.34'
-    exit 0
+
+set -e
```

Comment:
Do we need to re-generate the mariadb sources based on this change? Or is this script fix just for the next time the source verison is updated?

Reply:
yes, but I already did it